autoload/keepeye/system.vim

```diff
@@ -8,6 +8,10 @@ endfunction
 function! keepeye#system#Bell()
   if executable('mpv')
     call system('mpv ' . shellescape(g:keepeye_bell_path) . '&')
+  elseif executable('mocp')
+    call system('mocp -l ' . shellescape(g:keepeye_bell_path) . '&')
+  elseif executable('cvlc')
+    call system('cvlc ' . shellescape(g:keepeye_bell_path) . ' 2> /dev/null &')
```

Comment:
To easily add other players in the future, I suggest to put these players in a List, iterate over them, and stop on the first player available. What do you thing ?

Reply:
And why this error stream redirected to `/dev/null` ? I mean, is it really needed ?

Reply:
Hi @soywod.

First of all. I didn't have the chance yet so....Thanks for this plugin. It is pretty cool :)

Well. It's the first time I'm seeing the vim syntax and I don't know much about it.

The cvlc command produces some output when it runs so I though it would be a good idea to send this output to /dev/null. It works but I'm not sure if is needed to the plugin to work.

The list thing sounds like a better idea. So everyone could add a new player. It would be better if you could configure it. What do you think?

I have no idea how to do so but I would love to have a look and send you a new pull request.

Would we stay with the list of with a configurable parameter for the player?

And I'm going to update the docs as soon as we decide what to do next! \o/